wpilibNewCommands/src/test/native/cpp/frc2/command/SequentialCommandGroupTest.cpp

```diff
@@ -124,17 +124,20 @@ TEST_F(SequentialCommandGroupTest, SequentialGroupRequirement) {
   TestSubsystem requirement3;
   TestSubsystem requirement4;
 
-  InstantCommand command1([] {}, {&requirement1, &requirement2});
-  InstantCommand command2([] {}, {&requirement3});
-  InstantCommand command3([] {}, {&requirement3, &requirement4});
+  auto command1 = cmd::RunOnce([] {}, {&requirement1, &requirement2});
+  // InstantCommand command1([] {}, {&requirement1, &requirement2});
+  auto command2 = cmd::RunOnce([] {}, {&requirement3});
+  auto command3 = cmd::RunOnce([] {}, {&requirement3, &requirement4});
 
-  SequentialCommandGroup group(std::move(command1), std::move(command2));
+  // SequentialCommandGroup group(std::move(command1), std::move(command2));
+  auto group = cmd::Sequence(
+      make_vector<CommandPtr>(std::move(command1), std::move(command2)));
```

Comment:
```suggestion
  auto group = cmd::Sequence(std::move(command1), std::move(command2));
```
I think this would work?